cmd/fyne/internal/commands/build.go

```diff
@@ -202,6 +202,12 @@ func (b *Builder) build() error {
 		versionConstraint = version.NewConstrainGroupFromString(">=1.17")
 		env = append(env, "GOARCH=wasm")
 		env = append(env, "GOOS=js")
+	} else if goos == "gopherjs" {
+		_, err := b.runner.runOutput("version")
```

Comment:
is version needed? I would have thought that `exec.LookPath` would be faster

Reply:
It would be faster, but doesn't guarantee a working gopherjs. I did think that executing version would give more guarantee. If we are fine with a exec.LookPath, I can implement that.